app/services/cfe/service_set.rb

```diff
@@ -40,10 +56,26 @@ def initialize(object)
     end
 
     def call
+      sets[key]
+    end
```

Comment:
I'm not suggesting we make this change in this PR, but I'm not entirely comfortable with the interface here. 

To me at least, I feel like we could reveal more intent than we are currently. We're working in a complex domain already, so any place we can reduce cognitive load feels like a win!

This class is called in one place: `app/services/cfe/submission_manager.rb:34`
```ruby
def services
  ServiceSet.call(submission)
end
```

I wonder whether we could refactor this `ServiceSet` class slightly. 

```ruby
module CFE
  class ServiceSet
    def self.for(submission)
      new(submission).services
    end
    
    def services
      services_for_submission
    end
    
    private
    
    def services_for_submission
      if submission.passported?
        PASSPORTED_SERVICES
      elsif ...
        ...
      end
    end
  end
end
```

This would address a couple of things I found myself thinking when I read the code:
  - what does `object` actually refer to?
  - what does `call` actually do?
  
Ultimately this isn't a big deal, and perhaps we're wedded to the `ServiceObject/.call` pattern —but it's just some thoughts I had 😄

Reply:
yeah, lets simplify for now

Reply:
but the point of `object` is to highlight the decoupled nature of this class. This would work for an application object too